src/codeBuilders.js

```diff
@@ -41,3 +41,20 @@ exports.buildMethodCode = function(methodName, defaultPublicPath, shouldSupressE
     "}"
   ].join("\n");
 };
+
+
+exports.buildVariableCode = function (variableName, defaultPublicPath, shouldSupressErrors = false) {
+  return [
+    "try {",
+    `  if (typeof ${variableName} !== "string") {`,
+    `    throw new Error("${PLUGIN_NAME}: '${variableName}' is not a string or not available at runtime. See https://github.com/agoldis/webpack-require-from#troubleshooting");`,
+    "  }",
+    `  return (function(){if($PIX) return ${variableName};})();`,
```

Comment:
`$PIX`?

Reply:
`$PIX` is the global object in my project, I committed this by mistake, will remove it.